types/react-tag-input/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "baseUrl": "../",
+        "typeRoots": [
+            "../"
+        ],
+        "types": [],
+        "noEmit": true,
+        "forceConsistentCasingInFileNames": true
+    },
+    "files": [
+        "index.d.ts",
+        "react-tag-input-tests.ts"
```

Comment:
This should be `react-tag-input-tests.tsx` and add a `"jsx": "preserver"` as a compiler option. The remaining issues should be easy to fix.